etc/disable-interpreters.inc

```diff
@@ -12,10 +12,12 @@ blacklist /usr/lib64/libgjs*
 
 # Lua
 blacklist ${PATH}/lua*
+blacklist /usr/include/lauxlib.h
```

Comment:
`/etc/firejail/disable-devel.inc:blacklist /usr/include`

Reply:
Should be fixed now, thanks.